hosts

```diff
@@ -49,6 +49,7 @@ lae1.princeton.edu
 lae2.princeton.edu
```

Comment:
If you rebase this PR, be sure to get rid of these changes - inventory is in `inventory/` since #3258 was merged.